### DISCUSSION TITLE
Observing deviating drag coefficient values in the confined cylinder case in of90 and the corresponding chapter in the Rheotool userguide

### DISCUSSION THREAD

**Discussion by Flo1314** [fppimenta/rheoTool#112] [General]
Hello, I would like to compute the drag coefficient for a confined cylinder case and a custom solver that I am working on. Therefore, I would like to use the of90/tutorials/rheoFoam/Cylinder/Oldroyd-BLog case as a basis.

Unfortunately, I just realized that the Cd value for this case (the unchanged of90/tutorials/rheoFoam/Cylinder/Oldroyd-BLog case) converges to a value of approximately 117.646. However, the Rheotool Userguide (Chapter 5.1.6. - Results) states that a Cd value of 117.357 should be reached with this setup.

I would like to know if there are any special configurations that have to be enabled in order to reach this value of 117.357?

**Comment by fppimenta** [comment]
https://github.com/fppimenta/rheoTool/issues/36

**Reply by Flo1314** [reply]
Thank you very much!
